docs/all.md

```diff
@@ -2,6 +2,24 @@
 
 This documentation is meant to be applicable generally to all resources the operator manages.
 
+## Table of Contents
+
+- [Chia Configuration](#chia-configuration)
+  - [Network Selection](#selecting-a-network)
```

Comment:
```suggestion
- [Network Selection](#selecting-a-network)
```

Reply:
I think I prefer it with the indentation. Thoughts on why they're better unindented?